src/services/TestDataService.ts

```diff
@@ -1111,6 +1112,22 @@ export class TestDataService {
         return salesChannel;
     }
 
+    async addSalesChannelLanguage(salesChannelId: string, languageCode: string) {
+        const language = await getLanguageData(languageCode, this.AdminApiClient);
```

Comment:
Should we really call this function here? I get that it's nice to only have to know the isoCode, but in my scenario at least, it would mean I'd have to call this function more than once because I need the languageId for other things, too.

Reply:
I don't get your expectation, maybe share your scenario here would help?

Let me explain why using the languageCode here: What I need is the language.id, but it can be dynamic because it's an id. On the other hand, the isoCode does not change that frequently, so I can base on the code to get whatever id it will be at the moment the test is executing.

Reply:
So in my scenario I need to:

1) Assign a language to a sales channel
2) Create a Sales Channel domain with that new language

For both steps I need the `languageId`. So for me it makes the most sense to write something like:

```
const language = getLanguageData(languageCode);
const languageId = language.id;

await TestDataService.assignSalesChannelLanguage(salesChannelId, languageId);
await TestDataService.createSalesChannelDomain(salesChannelId, languageId, x, y, z);
```

So in my case, I'd end up calling `getLanguageData()` at least twice because currently, assignSalesChannelLanguage calls it again because it asks for the language isoCode and not the languageId. For a similar case, if you look at `assignSalesChannelCurrency()` you'll see that `getCurrency()` is a separate call.